### PR TITLE
chore: add python 3.10 classifier

### PR DESCRIPTION
**Pull request by anukaal**
added support for python 3.10

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-aiplatform/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -111,6 +111,7 @@
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
```
Comment (parthea) [inline setup.py:114]: Python 3.10 is not currently enabled in unit tests. This line also needs to be updated to include python 3.10: https://github.com/googleapis/python-aiplatform/blob/main/owlbot.py#L87

**Comment by parthea** [comment]
The python 3.10 is failing because the latest stable release of tensorflow 2.7.0 doesn't support python 3.10.
A pre-release version that supports python 3.10 is available: https://pypi.org/project/tensorflow/2.8.0rc0/ . Perhaps we should wait for a stable version.


```
nox > Running session unit-3.10
nox > Creating virtual environment (virtualenv) using python3.10 in .nox/unit-3-10
nox > python -m pip install mock asyncmock pytest pytest-cov pytest-asyncio -c /tmpfs/src/github/python-aiplatform/testing/constraints-3.10.txt
nox > python -m pip install -e '.[testing]' -c /tmpfs/src/github/python-aiplatform/testing/constraints-3.10.txt
nox > Command python -m pip install -e '.[testing]' -c /tmpfs/src/github/python-aiplatform/testing/constraints-3.10.txt failed with exit code 1:
Obtaining file:///tmpfs/src/github/python-aiplatform
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting google-api-core[grpc]<3.0.0dev,>=1.26.0
  Using cached google_api_core-2.4.0-py2.py3-none-any.whl (111 kB)
Collecting proto-plus>=1.10.1
  Using cached proto_plus-1.19.8-py3-none-any.whl (45 kB)
Requirement already satisfied: packaging>=14.3 in ./.nox/unit-3-10/lib/python3.10/site-packages (from google-cloud-aiplatform==1.9.0) (21.3)
Collecting google-cloud-storage<2.0.0dev,>=1.32.0
  Using cached google_cloud_storage-1.44.0-py2.py3-none-any.whl (106 kB)
Collecting google-cloud-bigquery<3.0.0dev,>=1.15.0
  Using cached google_cloud_bigquery-2.32.0-py2.py3-none-any.whl (205 kB)
Collecting lit-nlp>=0.4.0
  Using cached lit_nlp-0.4.1-py3-none-any.whl (746 kB)
ERROR: Could not find a version that satisfies the requirement tensorflow>=2.3.0 (from google-cloud-aiplatform[testing]) (from versions: 2.8.0rc0)
ERROR: No matching distribution found for tensorflow>=2.3.0
```

**Comment by parthea** [comment]
FYI @vinnysenthil

**Comment by sasha-gitg** [comment]
Closing in favor of https://github.com/googleapis/python-aiplatform/pull/1644

**Pull request closed without merging**